sqs/subscriber.go

```diff
@@ -94,33 +98,41 @@ func (s *Subscriber) receive(ctx context.Context, queueURL string, output chan *
 		"queue":    queueURL,
 	}
 
+	go func() {
```

Comment:
without that tests were hanging - I added err check for all places that may return canceled ctx after that